src/react/account/CreateAccountPolicy.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { useForm } from 'react-hook-form';
+import FormContainer, * as F from '../ui-elements/FormLayout';
+import { SecondaryText  } from '@scality/core-ui/dist/components/text/Text.component';
+import CopyButton from '../ui-elements/CopyButton';
+import { useMutation  } from 'react-query';
+import { getListPoliciesQuery } from '../queries';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useIAMClient } from '../IAMProvider';
+import { queryClient } from '../App';
+import { useHistory, useParams } from 'react-router-dom';
+
+type FormValues = {
+  policyName: string;
+  policyDocument: string;
+};
+
+type Props = {
+  accountName: string;
+  policyName: string;
+  policyDocument: string;
+}
+
+const CreateAccountPolicy = () => {
+  const IAMClient = useIAMClient();
+  const history = useHistory();
+  const { accountName } = useParams();
+
+  const defaultValues = {
+    policyName: '',
+    policyDocument: '',
+  };
+  const {
+    register,
+    reset,
+    handleSubmit,
+    watch,
+    formState: { isDirty, errors,  },
+  } = useForm<FormValues>({
+    defaultValues
+  });
+
+  const watchAllFields = watch();
+
+  const createPolicyMutation = useMutation(
+    ({ policyName, policyDocument } : Props) => {
```

Comment:
would be more straightforward 
```suggestion
    ({ policyName, policyDocument } : {policyName:string, policyDocument:string }) => {
```